Essentials/src/com/earth2me/essentials/User.java

```diff
@@ -25,13 +25,7 @@
 import org.bukkit.potion.PotionEffectType;
 
 import java.math.BigDecimal;
-import java.util.Calendar;
-import java.util.GregorianCalendar;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.UUID;
-import java.util.WeakHashMap;
+import java.util.*;
```

Comment:
Don't use `*` imports - I know IntelliJ does it automatically but you should turn it off in this case